packages/flutter_tizen/lib/src/rendering/platform_view.dart

```diff
@@ -127,6 +129,17 @@ class RenderTizenView extends PlatformViewRenderBox {
     markNeedsPaint();
   }
 
+  void _setOffset() {
+    SchedulerBinding.instance.addPostFrameCallback((_) async {
+      if (!_isDisposed) {
+        if (attached) {
+          await _viewController.setOffset(localToGlobal(Offset.zero));
+        }
+        _setOffset();
```

Comment:
I can't remember, but I think it's because it's basically based on the platform view that was in our web view implementation. so at the time, it seems like no effort was made to copy the comments from the original Flutter implementation.